### PR TITLE
example/networking/net_monitor.py added

### PR DESCRIPTION
**Pull request by gist-banana**
net_interface_network_stat_monitor.py gives you integrated data of how many packets were sent from a source_ip-destinattion_ip value pair.
For usage, open the code and edit the INTERFACE to the network interface name you wish to apply.

### CODE REVIEW

**Comment by yonghong-song** [comment]
[buildbot, test this please]

File: examples/tracing/net_interface_network_stat_monitor.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        for i in range(0,output_len):
+            if (len(str(packet_cnt_output[i][0]))) != 30:
+                continue
+            temp = int(str(packet_cnt_output[i][0])[8:-2]) # initial output omitted from the kernel space program
+            temp = int(str(bin(temp))[2:]) # raw file
+            src = int(str(temp)[:32],2) # part1 
+            dst = int(str(temp)[32:],2)
+            pkt_num = str(packet_cnt_output[i][1])[7:-1]
+
+            monitor_result = 'source address : ' + decimal_to_human(str(src)) + ' ' + 'destination address : ' + decimal_to_human(str(dst)) + ' ' + pkt_num + ' ' + 'time : ' + str(time.localtime()[0])+';'+str(time.localtime()[1]).zfill(2)+';'+str(time.localtime()[2]).zfill(2)+';'+str(time.localtime()[3]).zfill(2)+';'+str(time.localtime()[4]).zfill(2)+';'+str(time.localtime()[5]).zfill(2)
```
Comment (yonghong-song) [inline examples/tracing/net_interface_network_stat_monitor.py]: The line is too long. Let us try to keep it within 80 char's per line.

File: examples/tracing/net_interface_network_stat_monitor.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    u8 *cursor = 0;
+    u32 saddr;
+    u32 daddr;
+    long* count = 0;
+    long one = 1;
+    u64 pass_value = 0;
+    
+    struct ethernet_t *ethernet = cursor_advance(cursor, sizeof(*ethernet));
+    if (!(ethernet -> type == 0x0800)) {    
+        return 0; // drop
```
Comment (yonghong-song) [inline examples/tracing/net_interface_network_stat_monitor.py]: This a packet monitor, you do not want to drop the packet, right?

File: examples/tracing/net_interface_network_stat_monitor.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+BPF_PERF_OUTPUT(skb_events);
+BPF_HASH(packet_cnt, u64, long, 256); 
+
+int packet_monitor(struct __sk_buff *skb) {
+    u8 *cursor = 0;
+    u32 saddr;
+    u32 daddr;
+    long* count = 0;
+    long one = 1;
+    u64 pass_value = 0;
```
Comment (yonghong-song) [inline examples/tracing/net_interface_network_stat_monitor.py]: Maybe consolidate declarations into fewer lines.

File: examples/tracing/net_interface_network_stat_monitor.py [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    input_value = int(input_value)
+    hex_value = hex(input_value)[2:]
+    pt3 = literal_eval((str('0x'+str(hex_value[-2:]))))
+    pt2 = literal_eval((str('0x'+str(hex_value[-4:-2]))))
+    pt1 = literal_eval((str('0x'+str(hex_value[-6:-4]))))
+    pt0 = literal_eval((str('0x'+str(hex_value[-8:-6]))))
+    result = str(pt0)+'.'+str(pt1)+'.'+str(pt2)+'.'+str(pt3)
+    return result
+
+print("=========================packet monitor=============================\n")
```
Comment (yonghong-song) [inline examples/tracing/net_interface_network_stat_monitor.py]: You probably do not need this.

**Comment by gist-banana** [comment]
> Currently, you put the file under examples/tracing/.... This is more like a networking tool. Please put it under examples/networking.
> 
> Also please rebase on top of latest master.

I have edited the codes to reflect your comments from your review!
I was trying to 'rebase on top of latest master', so I did : 
'git rebase' master and it says 'Current branch master is up to date'.
So I believe it is rebased on latest master or I'm missing something.. (sry, this is my first time going through the whole pull request process)
I will re-request a review.
Thank you!

**Comment by yonghong-song** [comment]
[buildbot, test this please]

**Pull request merged**